### DISCUSSION TITLE
Field Validation for Read permissions

### DISCUSSION THREAD

**Discussion by sierses** [directus/directus#16383] [Feature Requests]
I'm looking for a way to only return results when certain conditions are met for the public role to control how the API is accessed.
Field Validation options are currently available for Update and Create.
Making it also available for Read would reduce the need to write extensions for simple validations.

**Comment by rijkvanzanten** [comment]
What kind of conditions? Sounds like regular permissions to me 🙂

**Reply by sierses** [reply]
In my case, to only return values when multiple fields fields are searched for.
I can't find a way to do it with regular permissions.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
